### PR TITLE
Replace all dots in name

### PR DESCRIPTION
**Pull request by tenshiak**
Right now it replaces just the first one

### CODE REVIEW

File: src/jackett.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function reformatTitleForSearching(name) {
 		: movieMatch
 		? movieMatch[0]
 		: name;
-	return fullMatch.replace(".", " ");
+	return fullMatch.replace(/\./g, " ");
```
Comment (mmgoodnow) [inline src/jackett.js:19]: Good catch! Have you tested this? I'm not sure the `replace` method will do that. 
```suggestion
	return fullMatch.replaceAll(".", " ");
```

Comment (tenshiak) [reply src/jackett.js:19]: I tested my version, yes. Not sure about replaceAll, I'm not a JS dev. Looks like replaceAll is a prototype function.

Comment (mmgoodnow) [reply src/jackett.js:19]: They're both prototype functions :) If it works, I'm satisfied.

**Review by mmgoodnow — APPROVED** [review]

**Comment by mmgoodnow** [comment]
I'll merge and release this later today. Thanks for the contribution!

**Pull request merged**